app/build.gradle

```diff
@@ -52,10 +52,10 @@ if (file('signing.gradle').exists()) {
 
 android {
     defaultConfig {
-        compileSdk 34
+        compileSdkPreview = "VanillaIceCream"
```

Comment:
Could you use API level rather than the name? aka: 35

Reply:
No according to [this docs](https://developer.android.com/codelabs/edge-to-edge#2)
Since the stable release hasn't being published yet I use the `compileSdkPreview`. If not, edge to edge is not enable for default
![Screenshot 2024-09-18 at 6 44 08 p m](https://github.com/user-attachments/assets/37e25893-5f1c-42b7-8481-c62dba1528c7)